cognite/seismic/protos/v1/seismic_service_messages.proto

```diff
@@ -226,3 +227,49 @@ message SegYSeismicResponse {
 message SearchFilesRequest {
     SearchSpec spec = 1;
 }
+
+message SearchJobStatusRequest {
+    message Filter {
+        message Id {
+            oneof id {
+                string job_id = 1;
+                string file_id = 2;
+                string file_name = 3;
```

Comment:
Maybe we can do without the file name here? Leaving the relationship between file ids and names to the File management APIs.

Reply:
Can do